.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Tauri Docs",
+  "runArgs": [],
+  "forwardPorts": [],
+  "mounts": [],
+  "build": {
+    "dockerfile": "Dockerfile",
+    "args": {}
+  },
+  "customizations": {
+    "settings": {
+      "extensions.installRecommended": true
+    },
+    "extensions": [

```

Comment:
Haven't tested that 🤔 I'll give it a try. Otherwise a functionality difference could be that what we have in .devcontainer gets installed for everyone like it or not (like the Astro extension) while the rest are just recommended. Or we could simply put all extensions in both, it's not *that* big a deal I feel to have just the list of extensions in two places since preferably the list of extensions should be kept fairly limited so the environment doesn't have a million tools only a handful of people even know what they do

Reply:
I put the list of extensions in both files now, it's a short list, I think it's fine

Reply:
So Codespaces (and maybe Gitpod) don't read from `.vscode` and need them separately defined, right?

Reply:
Not sure, they should afaik, but they might just not honor the request to install them by default so people have to manually do it from the recommended tab, not sure